api/src/main/proto/spine/protodata/events.proto

```diff
@@ -264,3 +265,15 @@ message RpcExited {
 
     RpcName rpc = 3;
 }
+
+// Emitted when a Protobuf dependency file is discovered.
```

Comment:
Is this about discovering an `import` in our code? I am not sure what do you mean by "discovered".